vocode/streaming/transcriber/deepgram_transcriber.py

```diff
@@ -51,7 +51,22 @@ async def run(self):
                 "Deepgram connection died, restarting, num_restarts: %s", restarts
             )
 
+    def mute(self):
+        self.logger.debug("Muting DeepgramTranscriber")
+        super().mute()
+
+    def unmute(self):
+        self.logger.debug("Unmuting DeepgramTranscriber")
+        super().unmute()
+
+
     def send_audio(self, chunk):
+        if self.is_muted:
+            self.logger.debug("Transcriber is muted, not sending audio. {} {}".format(self.audio_queue.qsize(), self.audio_queue.empty()))
+            while not self.audio_queue.empty():
+                _ = self.audio_queue.get_nowait()
+            return
```

Comment:
Uses is_muted property to determine whether audio should actually get added to audio_queue